README.md

```diff
@@ -23,16 +23,34 @@ The URL from which SonarQube will be downloaded, and the resulting directory nam
 
     sonar_mysql_username: sonar
     sonar_mysql_password: sonar
-    
+
     sonar_mysql_host: localhost
     sonar_mysql_port: "3306"
     sonar_mysql_database: sonar
-    
+
     sonar_mysql_allowed_hosts:
       - 127.0.0.1
       - ::1
       - localhost
 
+To install any Sonar plugins you require, define these variables:
```

Comment:
For my role docs, the description goes underneath the configuration—in this case, can you put the `JDBC settings for a connection`... line back above this new block, and rewrite the "Use these variables to install any Sonar plugins you require." line under the new block.